StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml

```diff
@@ -135,13 +121,16 @@
       <sliceName value="prescriptionTaskStatusReason" />
       <short value="Prescription task status reason" />
       <definition value="The status history within EPS is primarily a technical Status and the statusReason will reflect current clinical status of the order." />
-      <min value="0" />
       <type>
         <code value="Extension" />
         <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionStatusHistory" />
       </type>
       <mustSupport value="true" />
     </element>
+    <element id="MedicationRequest.extension:prescriptionTaskStatusReason.extension">
+      <path value="MedicationRequest.extension.extension" />
+      <min value="1" />
```

Comment:
Is this right to add min cardinality of 1?

Reply:
This is required as the extension is a complex extension and doesn't have a simple 'value', again recalculated by Forge and seems correct.